src/ddmd/doc.d

```diff
@@ -1643,7 +1643,13 @@ struct DocComment
                     const(char)* q = p + utfStride(p);
                     while (isIdTail(q))
                         q += utfStride(q);
-                    if (*q == ':') // identifier: ends it
+                    if (*q == ':' &&            // 'identifier:' ends it
+                                                // but not 'http://' or 'https://'
+                        !(q[1] == '/' && q[2] == '/' &&
+                          (q - p == 4 && Port.memicmp(p, "http".ptr, 4) == 0 ||
+                           q - p == 5 && Port.memicmp(p, "https".ptr, 5) == 0)
+                         )
+                       )
```

Comment:
It is, but setting up the arguments to `skippastURL` means conversion to OutBuffer, and this string isn't in an OutBuffer.